core/dbt/parser/generic_test_builders.py

```diff
@@ -180,6 +180,7 @@ def from_test_block(
         )
 
 
+# TODO: create a ConstraintsBuilder equivalent
 class TestBuilder(Generic[Testable]):
```

Comment:
I should NOT need to build a `ConstraintsBuilder` because the configs extend `column_info` within the `schemas.py` file.